client/webserver/site/src/js/wallets.js

```diff
@@ -237,6 +240,20 @@ export default class WalletsPage extends BasePage {
     await this.newWalletForm.loadDefaults()
   }
 
+  async rescanWallet (assetID) {
+    if (app().haveAssetOrders(assetID)) {
+      const asset = app().assets[assetID]
+      app().notify(ntfn.make('Cannot rescan wallet.', `There are active orders involving ${asset.info.name}`, ntfn.ERROR)) // TODO: translate
+      return // TODO: show override prompt
```

Comment:
![image](https://user-images.githubusercontent.com/9373513/150224111-cde30867-e199-48f6-a0ca-9fda4d7e6d48.png)

Users could still use dexcctl because that's more of a power user tool, but I don't want users clicking the "Rescan" button willy-nilly.  I'm thinking a form can open if there are active orders to allow the user to initiate the request anyway.